# Translations template for neutron.
# Copyright (C) 2014 ORGANIZATION
# This file is distributed under the same license as the neutron project.
#
# Translators:
msgid ""
msgstr ""
"Project-Id-Version: Neutron\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2014-09-29 06:04+0000\n"
"PO-Revision-Date: 2014-08-26 11:11+0000\n"
"Last-Translator: openstackjenkins <jenkins@openstack.org>\n"
"Language-Team: Portuguese (Brazil) (http://www.transifex.com/projects/p/"
"neutron/language/pt_BR/)\n"
"Language: pt_BR\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 1.3\n"
"Plural-Forms: nplurals=2; plural=(n > 1);\n"

#: neutron/policy.py:119
#, python-format
msgid ""
"Backward compatibility unavailable for deprecated policy %s. The policy will "
"not be enforced"
msgstr ""
"Retro compatibilidade não disponível para política deprecada %s. A política "
"não será garantida"

#: neutron/policy.py:311
#, python-format
msgid "Policy check error while calling %s!"
msgstr "Erro de verificação de políticas enquanto chamando %s!"

#: neutron/agent/l3_ha_agent.py:136
#, python-format
msgid "Unable to process HA router %s without ha port"
msgstr ""

#: neutron/agent/linux/external_process.py:229
#, python-format
msgid ""
"%(service)s for %(resource_type)s with uuid %(uuid)s not found. The process "
"should not have died"
msgstr ""

#: neutron/agent/linux/external_process.py:249
#, python-format
msgid "respawning %(service)s for uuid %(uuid)s"
msgstr ""

#: neutron/agent/linux/external_process.py:255
msgid "Exiting agent as programmed in check_child_processes_actions"
msgstr ""

#: neutron/agent/linux/interface.py:154
#, python-format
msgid "Failed deleting ingress connection state of floatingip %s"
msgstr ""

#: neutron/agent/linux/interface.py:163
#, python-format
msgid "Failed deleting egress connection state of floatingip %s"
msgstr ""

#: neutron/cmd/sanity_check.py:58
msgid ""
"Nova notifications are enabled, but novaclient is not installed. Either "
"disable nova notifications or install python-novaclient."
msgstr ""

#: neutron/openstack/common/excutils.py:76
#, python-format
msgid "Original exception being dropped: %s"
msgstr "Exceção original sendo descartada: %s"

#: neutron/openstack/common/excutils.py:105
#, python-format
msgid "Unexpected exception occurred %d time(s)... retrying."
msgstr "Exceção não esperada ocorreu %d vez(es)... tentando novamente."

#: neutron/openstack/common/periodic_task.py:202
#, python-format
msgid "Error during %(full_task_name)s: %(e)s"
msgstr "Erro durante %(full_task_name)s: %(e)s"

#: neutron/openstack/common/service.py:188
msgid "Exception during rpc cleanup."
msgstr "Exceção durante limpeza de RPC."

#: neutron/openstack/common/service.py:277
msgid "Unhandled exception"
msgstr "Exceção não tratada"

#: neutron/plugins/cisco/models/virt_phy_sw_v2.py:279
#, python-format
msgid "Unable to delete port '%(pname)s' on switch. Exception: %(exp)s"
msgstr ""

#: neutron/plugins/ml2/drivers/type_gre.py:86
#, python-format
msgid "Skipping unreasonable gre ID range %(tun_min)s:%(tun_max)s"
msgstr "Pulando faixa irracional do ID GRE %(tun_min)s:%(tun_max)s"

#: neutron/plugins/ml2/drivers/type_vxlan.py:92
#, python-format
msgid "Skipping unreasonable VXLAN VNI range %(tun_min)s:%(tun_max)s"
msgstr "Pulando faixa irracional do VXLAN VNI %(tun_min)s:%(tun_max)s"

#: neutron/plugins/ml2/drivers/cisco/apic/apic_topology.py:110
msgid "APIC service agent: failed in reporting state"
msgstr ""

#: neutron/plugins/ml2/drivers/cisco/apic/apic_topology.py:205
#, python-format
msgid "No such interface (ignored): %s"
msgstr ""

#: neutron/plugins/ml2/drivers/cisco/apic/apic_topology.py:255
msgid "APIC service agent: exception in LLDP parsing"
msgstr ""

#: neutron/plugins/ml2/drivers/cisco/apic/apic_topology.py:310
#, python-format
msgid "APIC service agent: can not get MACaddr for %s"
msgstr ""

#: neutron/plugins/ml2/drivers/cisco/apic/apic_topology.py:326
msgid "APIC host agent: failed in reporting state"
msgstr ""

#: neutron/plugins/mlnx/agent/eswitch_neutron_agent.py:386
msgid "Error in agent event loop"
msgstr "Erro no loop de eventos do agente"

#: neutron/plugins/nuage/syncmanager.py:55
#, python-format
msgid "Cannot complete the sync between Neutron and VSD because of error:%s"
msgstr ""

#: neutron/plugins/ofagent/agent/ofa_neutron_agent.py:102
msgid "Agent terminated!: Failed to get a datapath."
msgstr ""

#: neutron/plugins/ofagent/agent/ofa_neutron_agent.py:121
msgid "Agent terminated"
msgstr ""

#: neutron/plugins/ofagent/agent/ofa_neutron_agent.py:158
msgid "Agent failed to create agent config map"
msgstr ""

#: neutron/plugins/ofagent/agent/ofa_neutron_agent.py:283
msgid "Failed reporting state!"
msgstr ""

#: neutron/plugins/ofagent/agent/ofa_neutron_agent.py:449
#, python-format
msgid "No local VLAN available for net-id=%s"
msgstr ""

#: neutron/plugins/ofagent/agent/ofa_neutron_agent.py:464
#, python-format
msgid ""
"Cannot provision %(network_type)s network for net-id=%(net_uuid)s - "
"tunneling disabled"
msgstr ""

#: neutron/plugins/ofagent/agent/ofa_neutron_agent.py:475
#, python-format
msgid ""
"Cannot provision %(network_type)s network for net-id=%(net_uuid)s - no "
"bridge for physical_network %(physical_network)s"
msgstr ""

#: neutron/plugins/ofagent/agent/ofa_neutron_agent.py:485
#, python-format
msgid ""
"Cannot provision unknown network type %(network_type)s for net-id="
"%(net_uuid)s"
msgstr ""

#: neutron/plugins/ofagent/agent/ofa_neutron_agent.py:523
#, python-format
msgid ""
"Cannot reclaim unknown network type %(network_type)s for net-id=%(net_uuid)s"
msgstr ""

#: neutron/plugins/ofagent/agent/ofa_neutron_agent.py:657
#, python-format
msgid ""
"Bridge %(bridge)s for physical network %(physical_network)s does not exist. "
"Agent terminated!"
msgstr ""

#: neutron/plugins/ofagent/agent/ofa_neutron_agent.py:735
msgid "ofport should have a value that can be interpreted as an integer"
msgstr ""

#: neutron/plugins/ofagent/agent/ofa_neutron_agent.py:738
#, python-format
msgid "Failed to set-up %(type)s tunnel port to %(ip)s"
msgstr ""

#: neutron/plugins/ofagent/agent/ofa_neutron_agent.py:916
msgid "Error while synchronizing tunnels"
msgstr ""

#: neutron/plugins/ofagent/agent/ofa_neutron_agent.py:956
msgid "Error while processing VIF ports"
msgstr ""

#: neutron/plugins/vmware/plugins/base.py:2099
#, python-format
msgid ""
"Rolling back database changes for gateway device %s because of an error in "
"the NSX backend"
msgstr ""

#: neutron/scheduler/l3_agent_scheduler.py:281
#, python-format
msgid "Not enough candidates, a HA router needs at least %s agents"
msgstr ""

#: neutron/services/vpn/service_drivers/cisco_cfg_loader.py:62
#, python-format
msgid "Config file parse error: %s"
msgstr ""

#: neutron/services/vpn/service_drivers/cisco_cfg_loader.py:76
#, python-format
msgid "Ignoring Cisco CSR configuration entry - router IP %s is not valid"
msgstr ""

#: neutron/services/vpn/service_drivers/cisco_cfg_loader.py:89
#, python-format
msgid "Ignoring Cisco CSR for router %(router)s - missing %(field)s setting"
msgstr ""

#: neutron/services/vpn/service_drivers/cisco_cfg_loader.py:97
#, python-format
msgid ""
"Ignoring Cisco CSR for router %s - timeout is not a floating point number"
msgstr ""

#: neutron/services/vpn/service_drivers/cisco_cfg_loader.py:113
#, python-format
msgid "Ignoring Cisco CSR for router %s - local tunnel is not an IP address"
msgstr ""

#: neutron/services/vpn/service_drivers/cisco_cfg_loader.py:119
#, python-format
msgid "Malformed interface name for Cisco CSR router entry - %s"
msgstr ""

#: neutron/services/vpn/service_drivers/cisco_cfg_loader.py:155
#: neutron/services/vpn/service_drivers/cisco_cfg_loader.py:202
msgid "No routers found in INI file!"
msgstr ""

#: neutron/services/vpn/service_drivers/cisco_cfg_loader.py:174
#, python-format
msgid "Unable to lookup router ID based on router's public IP (%s) in INI file"
msgstr ""

#: neutron/services/vpn/service_drivers/cisco_cfg_loader.py:177
#, python-format
msgid "No matching routers on host %s"
msgstr ""

#: neutron/services/vpn/service_drivers/cisco_cfg_loader.py:211
#, python-format
msgid "Unable to find host for router %s"
msgstr ""
